### ISSUE TITLE
פיבלוש מפה - חלון חדש

### ISSUE THREAD

**Issue by emrib** [hasadna/datacity-mapali#18]
אחרי שמפבלשים את המפה היא נוצרת באותו חלון בדפדפן. פתיחה בחלון חדש תאפשר לעשות שינויים קטנים שלא עלו עליהם בתצוגה במקדימה ולפבלש שוב.

**Comment by LiorKovalio** [comment]
I would like to help with this issue.

The naive solution is simple, refactoring [map on status done](https://github.com/hasadna/datacity-mapali/blob/main/ui/src/app/map/map.component.ts#L48) with
```
-        this.router.navigate(['/m', this.lastId]);
+        const url = this.router.serializeUrl(this.router.createUrlTree(['/m', this.lastId]));
+        window.open(url, '_blank');
```
yet there are issues:
* This requires the user to allow for popup - is it acceptable?
* How should the original window function? The map should go back to status "ready"? The drawer should go back to "not processing", showing "מקור המידע" , "מידע כללי", "מיפויים" ?

**Comment by akariv** [comment]
Yes, requiring the user to allow pop-ups is not really a good solution.

I think for this to work we would have to show the user a message, saying "your map is ready", and then open it in a new tab using a regular link with target=blank.

**Comment by akariv** [comment]
Also sorry for the long response time 😞

**Comment by LiorKovalio** [comment]
Thanks for the response.

I've forked my suggestion [here](https://github.com/LiorKovalio/datacity-mapali) .
* Clicking the "looking good" button starts the processing (as it was), then it changes to 2 buttons, one for review (link with _blank) and the other for the routing (as it used to be) .
![image](https://user-images.githubusercontent.com/29506962/231199881-f67b8fa7-6901-4a40-b975-1d2302bf76a7.png)

* Changing the data and metadata after "looking good" was clicked didn't work well, as the datapackage wasn't updating. I suspect some caching issue, which is strange because it does work before clicking "looking good" - in any case, salting the get request fixed it.
* I don't like how the components disappear and reappear while processing. If this solution is fine, I would like to change the components to gray-out instead.
